cibyl/cli/parser.py

```diff
@@ -151,4 +151,5 @@ def extend(self, arguments: list, group_name: str, level: int = 0):
                     default=arg.default,
                     level=level)
         except argparse.ArgumentError:
+            import pdb;pdb.set_trace()
```

Comment:
in python3 you can use the breakpoint function for this, without needing to import anything. You can even configure to use other debuggers (personally I prefer ipdb) with the following env variable
`export PYTHONBREAKPOINT=ipdb.set_trace`

Reply:
Thanks. Old habits and VIM shortcuts :)